lib/Dist/Zilla/Plugin/InstallGuide.pm

```diff
@@ -63,6 +63,11 @@ Alternatively, if your CPAN shell is set up, you should just be able to do:
 ## Manual installation
 
 {{ $manual_installation }}
+
+The prerequisites of this distribution will also have to be installed manually. The
+prerequisites are listed in one of the files: MYMETA.yml or MYMETA.json generated by
```

Comment:
Put backticks around the filenames

Reply:
Hi @doherty

I am not sure what you mean, the endpoint is a plain text file, as for PR #14.

Please enlighten me

Reply:
I have added backticks as requested :-)